perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_def_titleBeforeKos.java

```diff
@@ -36,9 +36,9 @@ public void changedAttributeHook(PerunSessionImpl session, User user, Attribute
 		if (attribute.getValue() != null) {
 			Attribute titleBeforeVema;
```

Comment:
maybe the name of this variable should be changed to `titleBeforeDc2`?

Reply:
You are right, thank you. Fixed.